test/test.js

```diff
@@ -303,13 +303,27 @@ describe('Filter', function() {
     };
 
     beforeEach(function() {
-      f = new F(fixturePath, { persist: true });
+      this.originalCacheRoot = process.env.BROCCOLI_PERSISTENT_FILTER_CACHE_ROOT;
+    });
+
+    afterEach(function() {
+      process.env.BROCCOLI_PERSISTENT_FILTER_CACHE_ROOT = this.originalCacheRoot || '';
```

Comment:
sadly yes, because `process.env.something = undefined` actually sets the ENV var to the string `"undefined"`.